app/geant-exporter/GeantPhysicsTableWriter.cc

```diff
@@ -0,0 +1,305 @@
+        }
+
+        if (G4PhysicsTable* table = eloss_process.IonisationTable())
+        {
+            this->fill_tables_tree(*table, "Ionisation", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.IonisationTableForSubsec())
+        {
+            this->fill_tables_tree(
+                *table, "SubIonisation", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.CSDARangeTable())
+        {
+            this->fill_tables_tree(*table, "CSDARange", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.SecondaryRangeTable())
+        {
+            this->fill_tables_tree(*table, "RangeSec", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.RangeTableForLoss())
+        {
+            this->fill_tables_tree(*table, "Range", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.InverseRangeTable())
+        {
+            this->fill_tables_tree(*table, "InverseRange", table_name, "eloss");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.LambdaTable())
+        {
+            this->fill_tables_tree(*table, "Lambda", table_name, "xs");
+        }
+
+        if (G4PhysicsTable* table = eloss_process.SubLambdaTable())
+        {
+            this->fill_tables_tree(*table, "SubLambda", table_name, "xs");
+        }
+    }
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Write multiple scattering tables to the TTree
+ */
+void GeantPhysicsTableWriter::fill_multiple_scattering_tables(
+    const G4VMultipleScattering& msc_process)
+{
+    std::string process_name = msc_process.GetProcessName();
+
+    // TODO: Figure out a method to get the number of models. Max is 4.
+    // Other classes have a NumberOfModels(), but not G4VMultipleScattering
+    for (auto i : celeritas::range(4))
+    {
+        if (G4VEmModel* model = msc_process.GetModelByIndex(i))
+        {
+            if (G4PhysicsTable* table = model->GetCrossSectionTable())
+            {
+                // Table type
+                std::string model_name = model->GetName();
+
+                // Write table_
+                table_.model = to_geant_model(model_name);
+
+                std::string table_name      = process_name + "_" + model_name;
+                std::string table_type_name = "LambdaMod"
+                                              + std::to_string(i + 1);
+                this->fill_tables_tree(
+                    *table, table_type_name, table_name, "eloss");
+            }
+        }
+    }
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Write a given G4PhysicsTable as a GeantPhysicsVector in this->table_
+ */
+void GeantPhysicsTableWriter::fill_physics_vectors(G4PhysicsTable& table,
+                                                   std::string     xs_or_eloss)
```

Comment:
I didn't see that this string was only ever `"xs"` or `"eloss"`. Avoid using "magic strings" or magic numbers and use enums instead. So change this to a class enum (can be private)
```c++
enum class VectorType {
    xs,
    eloss
};
```

Reply:
Actually make it public so you can change `geant_physics_vector.is_eloss` to the enum instead of a bool.